src/osd/PG.cc

```diff
@@ -213,7 +210,6 @@ PG::PG(OSDService *o, OSDMapRef curmap,
   pg_stats_publish_valid(false),
   finish_sync_event(NULL),
   scrub_after_recovery(false),
-  save_req_scrub(false),
```

Comment:
Removing this flag is likely causing TEST_recovery_scrub in qa/standalone/scrub/osd-recovery-scrub.sh to fail.

Reply:
That test failure is unrelated to save_req_scrub, but the failure of that test is not present in the current code.